driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverCapabilities.java

```diff
@@ -66,6 +66,7 @@ static void addGlobalCapability(String key, Object value) {
         GLOBALCAPABILITIES.put(key, value);
     }
 
+    @Deprecated
     static void addThreadCapability(String key, Object value) {
```

Comment:
You should add a note to use DesktopWebdriverRequset instead.

Reply:
I made it package private instead.